openjdk.test.jck/src/native/makefile

```diff
@@ -277,14 +277,10 @@ ifneq ($(OS),win)
 	$(COPYDIR) $(SRCDIR)$(D)tests$(D)api$(D)javax_management$(D)loading$(D)data$(D)* $(FULLOUTDIR)$(D).
 	cd $(FULLOUTDIR) && $(JAR) uf $(FULLOUTDIR)/archives/MBeanUseNativeLib.jar $(LIBPREF)systemInfo.$(LIBEXT)
 	cd $(FULLOUTDIR) && $(JAR) cf $(FULLOUTDIR)/archives/OnlyLibs.jar $(LIBPREF)jmxlibid.$(LIBEXT)
-	$(CLEANFILE) $(FULLOUTDIR)$(D)$(LIBPREF)systemInfo.$(LIBEXT) 
-	$(CLEANFILE) $(FULLOUTDIR)$(D)$(LIBPREF)jmxlibid.$(LIBEXT) 
```

Comment:
Are we leaving some build material behind without proper cleanup?
It appears that `$(JAR)` didn't finished successfully before the `$(CLEANFILE)` proceeds  particularly for `AIX`. Is there a way to check if  `MBeanUseNativeLib.jar` & `OnlyLibs.jar` have been created properly before `$(CLEANFILE)`? A simple thought is to check/sleep/retry three times before skipping `$(CLEANFILE)` at last.

Reply:
>Are we leaving some build material behind without proper cleanup?

We do not have to worry about it in the makefile any more since we have recently added a [step](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/jck/build.xml#L121) to clean up the entire 'natives' directory (in which all native libs get generated) from JCK root at the beginning of the jck compilation.

These  $(CLEANFILE) used to exist from before and they do not need to be there any more. 

>It appears that $(JAR) didn't finished successfully.. 

They are getting generated successfully , I ran an `ls -la` on the archive directory of the jck root on the machine in which the original test failed: https://hyc-runtimes-jenkins.swg-devops.com/view/Sandbox/job/all-nodes-matching-labels/266/console.

Reply:
> We do not have to worry about it in the makefile any more since we have recently added a step to clean up the entire 'natives' directory (in which all native libs get generated) from JCK root at the beginning of the jck compilation.

Okay, this works as well.
I think there was a time issue in `AIX` platform which leads a problematic package creation and caused `UnsatisfiedLinkError: Can't load jmxlibid` later.